docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     image: dashpay/dashd:${CORE_VERSION:-latest}
     restart: always
     ports:
-      - 20001:20001 # P2P
+      - ${P2P_PORT}:${P2P_PORT} # P2P
```

Comment:
Let's keep internal as is then you can always use the same port to connect in other services.

Reply:
OK, so internally we always use 20001 for P2P and 20002 for RPC? Are you sure this will work for mainnet masternodes? I remember there was a check to ensure port 9999 is used at some point, but if it is only during `register_prepare` then we will be specifying the external port there anyway.

Reply:
Hm.. I think you right. It’s better to set right P2P port to Dash Core. Anyways, we are not using P2P port in other services atm.